### PR TITLE
Fixed the documentation in the entire application

### PR DESCRIPTION
**Pull request by Te4-Alexander-Nilsson-Melbostad**
## What issue did you implement or fix?
Fixes #253 

## Summary
- Fixed the documentation in the entire application so that should clarify some things

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)
expected)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] New and existing unit tests pass locally with my changes

## How can this implementation be improved?
- change some documentation based on the feedback from the pull request maybe

### CODE REVIEW

File: backend/app/controllers/api/v1/titles_controller.rb [hunk 1]
```diff
@@ -1,14 +1,26 @@
+# THIS CONTROLLER IS NOT BEING AUTHORISED WITH PUNDIT
+
 class Api::V1::TitlesController < ApplicationController
   before_action :set_title, only: [:show, :update, :destroy]
 
+  # This is the index method toe render a JSON object with all titles
```
Comment (TE4-Mattis-Abrahamsson) [inline backend/app/controllers/api/v1/titles_controller.rb]: to*

File: backend/app/controllers/api/v1/titles_controller.rb [hunk 2]
```diff
@@ -1,14 +1,26 @@
...
 
+  # This is the index method toe render a JSON object with all titles 
+  # which shows all titles available.
+  # This specific controller is not being authorized
+
   def index
     render json: Title.all
   end
 
+  # This is the show method and also generates the Json object of instance of the class
```
Comment (TE4-Mattis-Abrahamsson) [inline backend/app/controllers/api/v1/titles_controller.rb]: JSON*

File: frontend/src/components/CigCanvas.vue [hunk 3]
```diff
@@ -36,6 +36,13 @@
   </div>
 </template>
 
+<!-- 
+  There are some uncommon imports in this file such as JQuery, resize and JsBarcode. Jquery
```
Comment (TE4-Mattis-Abrahamsson) [inline frontend/src/components/CigCanvas.vue]: jQuery is not used and should be removed asap

File: backend/app/controllers/api/v1/books_controller.rb [hunk 4]
```diff
@@ -1,14 +1,22 @@
+# THIS CONTROLLER IS NOT BEING AUTHORISED WITH PUNDIT
+
 class Api::V1::BooksController < ApplicationController
   before_action :set_book, only: [:show, :update, :destroy]
 
+  # index method, gets all instances of book and renders it
```
Comment (TE4-oskar-pilborg) [inline backend/app/controllers/api/v1/books_controller.rb]: To be consistent you should add a new line after this comment

File: backend/app/controllers/api/v1/books_controller.rb [hunk 5]
```diff
@@ -1,14 +1,22 @@
...
     render json: Book.all
   end
+
+  # show method, gets specific instance of book and renders it
   
   def show
     render json: @book
   end
 
+  # This is the create method simply outputting the instance of the json object @book based
```
Comment (TE4-oskar-pilborg) [inline backend/app/controllers/api/v1/books_controller.rb]: Remove "This is" or change so it is grammatically correct

File: backend/app/controllers/api/v1/books_controller.rb [hunk 6]
```diff
@@ -18,6 +26,9 @@ def create
     end
   end
 
+  # This is the update method simply outputting the instance of the json object @book based
```
Comment (TE4-oskar-pilborg) [inline backend/app/controllers/api/v1/books_controller.rb]: See above

File: backend/app/controllers/api/v1/books_controller.rb [hunk 7]
```diff
@@ -26,10 +37,16 @@ def update
...
   end
 
+  # Destroy method, destroys the single target instance of book and then renders the
+  # page with remaining instances
+
   def destroy
     @book.destroy
   end
 
+  # this is the private set book method to find the current instance of
```
Comment (TE4-oskar-pilborg) [inline backend/app/controllers/api/v1/books_controller.rb]: See above

File: backend/app/controllers/api/v1/books_controller.rb [hunk 8]
```diff
@@ -26,10 +37,16 @@ def update
...
+
   def destroy
     @book.destroy
   end
 
+  # this is the private set book method to find the current instance of 
+  # book based on the given id
+
   private
     # Use callbacks to share common setup or constraints between actions.
```
Comment (TE4-oskar-pilborg) [inline backend/app/controllers/api/v1/books_controller.rb]: New line after comment

File: backend/app/controllers/api/v1/titles_controller.rb (hunk 2 above)
Comment (TE4-oskar-pilborg) [inline backend/app/controllers/api/v1/titles_controller.rb]: and it* also

File: backend/app/controllers/application_controller.rb [hunk 9]
```diff
@@ -1,19 +1,30 @@
...
+
 class ApplicationController < ActionController::Base
   include Pundit
   protect_from_forgery
 
+  # Used as standard rescue method for method within Pundit that return false 
+
   rescue_from Pundit::NotAuthorizedError, with: :user_not_authorized
 
+  # this method sets the current user based on sessions user id or if the environment is test
```
Comment (TE4-oskar-pilborg) [inline backend/app/controllers/application_controller.rb]: This*

File: backend/app/controllers/application_controller.rb [hunk 10]
```diff
@@ -1,19 +1,30 @@
...
+  # sets the current user to the first of the instance users
```
Comment (TE4-oskar-pilborg) [inline backend/app/controllers/application_controller.rb]: Sets*

File: backend/app/controllers/application_controller.rb [hunk 11]
```diff
@@ -1,19 +1,30 @@
...
+
   def current_user
     return User.first if Rails.env.test?
     @_current_user ||= session[:current_user_id] &&
     User.find_by(uid: session[:current_user_id])
   end
 
+  # show basically renders the standard app html file
```
Comment (TE4-oskar-pilborg) [inline backend/app/controllers/application_controller.rb]: Show*

File: frontend/src/components/BookFormComponent.vue [hunk 12]
```diff
@@ -46,21 +46,33 @@ import TitlesModule from '../store/modules/TitlesModule';
 import BooksModule from '../store/modules/BooksModule';
 import { VuexModule } from 'vuex-module-decorators';
 
+// This is the component for the book form which includes all the information for the book
```
Comment (TE4-oskar-pilborg) [inline frontend/src/components/BookFormComponent.vue]: Bookform*

File: frontend/src/components/BookFormComponent.vue [hunk 13]
```diff
@@ -46,21 +46,33 @@ import TitlesModule from '../store/modules/TitlesModule';
...
 @Component
 export default class BookFormComponent extends Vue {
   private form: {} = {
     barcode: '',
     title_id: 0,
     status: '',
   };
+
+  // This is defining the title module as a module in order to use the module in the vue
+  // components
```
Comment (TE4-oskar-pilborg) [inline frontend/src/components/BookFormComponent.vue]: New line after comment

File: frontend/src/components/CanvasContainer.vue [hunk 14]
```diff
@@ -29,16 +29,26 @@ import JSZip from 'jszip';
     CigCanvas,
   },
 })
+
+// This is the canvas container component for the cards for the users (students)
+
+
```
Comment (TE4-oskar-pilborg) [inline frontend/src/components/CanvasContainer.vue]: Remove empty line

File: frontend/src/components/CanvasContainer.vue [hunk 15]
```diff
@@ -29,16 +29,26 @@ import JSZip from 'jszip';
...
   public sendCanvas: boolean = false;
   public imageBlobs: any[] = [];
 
+// Eventlistener GetAllCanvases is simply used in order to fetch all the canvases. 
+
   public getAllCanvases() {
     this.sendCanvas = !this.sendCanvas;
   }
 
+// The onImageReceived mehthod takes an image and then compares it to the instance of images
```
Comment (TE4-oskar-pilborg) [inline frontend/src/components/CanvasContainer.vue]: method*

File: frontend/src/components/CardFormComponent.vue [hunk 16]
```diff
@@ -23,18 +23,30 @@ import { Component, Watch, Prop, Vue } from 'vue-property-decorator';
...
 import JSZip from 'jszip';
 
+// Component for the card form for the user to insert information about them such as
+// role, name and uid
+
 @Component
 export default class CardFormComponent extends Vue {
   public files: File[] = [];
 
+  // Constructor for instantiating the class and fetching all the users from the userModule
```
Comment (TE4-oskar-pilborg) [inline frontend/src/components/CardFormComponent.vue]: New line after comment

File: frontend/src/components/CardFormComponent.vue [hunk 17]
```diff
@@ -45,13 +57,21 @@ export default class CardFormComponent extends Vue {
     }
   }
 
+  // LoadFileData takes three arguments, a list of files, the index of the list of files
```
Comment (TE4-oskar-pilborg) [inline frontend/src/components/CardFormComponent.vue]: loadFileData*

File: frontend/src/components/CardFormComponent.vue [hunk 18]
```diff
@@ -66,6 +86,9 @@ export default class CardFormComponent extends Vue {
     }
   }
 
+  // extract files method takes a zip file and creates a new instance of the class JSZip
```
Comment (TE4-oskar-pilborg) [inline frontend/src/components/CardFormComponent.vue]: ExtractFiles*

File: frontend/src/components/CigCanvas.vue [hunk 19]
```diff
@@ -64,12 +71,17 @@ export default class CigCanvas extends Vue {
   public size: number = 1;
   public context: any = null;
 
+  // method userNames is used in order to filter out the users that are not deleted to verify
```
Comment (TE4-oskar-pilborg) [inline frontend/src/components/CigCanvas.vue]: Method*

File: frontend/src/components/CigCanvas.vue [hunk 20]
```diff
@@ -80,6 +92,9 @@ export default class CigCanvas extends Vue {
     }
   }
 
+  // the generate Canvas does exactly what its called and uses the resize dependency
```
Comment (TE4-oskar-pilborg) [inline frontend/src/components/CigCanvas.vue]: The*

File: frontend/src/components/CigCanvas.vue [hunk 21]
```diff
@@ -98,12 +113,17 @@ export default class CigCanvas extends Vue {
     }
   }
 
+  // sets the instance of the height and width to the client hight and width to standardise
```
Comment (TE4-oskar-pilborg) [inline frontend/src/components/CigCanvas.vue]: Sets*

File: frontend/src/components/CigCanvas.vue [hunk 22]
```diff
@@ -98,12 +113,17 @@ export default class CigCanvas extends Vue {
...
+  // the structure of the canvas
+
   public getCanvasContainerSize() {
     const canvasContainer: any = this.$refs.canvasContainer;
     this.width = canvasContainer.clientWidth;
     this.height = canvasContainer.clientHeight;
   }
 
+  // set canvas size basically does the same thing but sets the dimensions to 2d
```
Comment (TE4-oskar-pilborg) [inline frontend/src/components/CigCanvas.vue]: Set*

File: frontend/src/components/CigCanvas.vue [hunk 23]
```diff
@@ -112,6 +132,9 @@ export default class CigCanvas extends Vue {
     htmlCanvasElement.height = this.height;
   }
 
+  // the draw image instantiates new classes of the image based on the barcode and also
```
Comment (TE4-oskar-pilborg) [inline frontend/src/components/CigCanvas.vue]: The drawImages*

File: frontend/src/components/CigCanvas.vue [hunk 24]
```diff
@@ -169,6 +192,9 @@ export default class CigCanvas extends Vue {
     }, 200);
   }
 
+  // draw text method renders the text on the canvas in order to style it and also get
```
Comment (TE4-oskar-pilborg) [inline frontend/src/components/CigCanvas.vue]: drawText

File: frontend/src/components/CigCanvas.vue [hunk 25]
```diff
@@ -190,6 +216,11 @@ export default class CigCanvas extends Vue {
     );
   }
 
+  // this is a watch action that monitors the send the instance sendCanvas to see if the
```
Comment (TE4-oskar-pilborg) [inline frontend/src/components/CigCanvas.vue]: This*

File: frontend/src/components/CigCanvas.vue [hunk 26]
```diff
@@ -199,6 +230,9 @@ export default class CigCanvas extends Vue {
     });
   }
 
+  // the download canvas method is used to download and also save the donwloaded zip to
```
Comment (TE4-oskar-pilborg) [inline frontend/src/components/CigCanvas.vue]: The downloadCanvas

File: frontend/src/components/CigCanvas.vue [hunk 27]
```diff
@@ -222,11 +260,15 @@ export default class CigCanvas extends Vue {
...
 
+  // onNameInput is a getter to receive the information stored in the instance of checkUserData
+  // and the generate Canvas 
+
   public onNameInput() {
     this.checkUserData();
     this.generateCanvas();
   }
 
+  // this is also a getter to receive information simply from the generate canvas instance
```
Comment (TE4-oskar-pilborg) [inline frontend/src/components/CigCanvas.vue]: This* and a new line after comment

File: frontend/src/components/CigCanvas.vue (hunk 3 above)
Comment (Te4-Alexander-Nilsson-Melbostad) [reply frontend/src/components/CigCanvas.vue]: I added documentiation about is since it was imported in the state that I was in at the time, but it is removed now!

File: frontend/src/components/GoogleLogin.vue [hunk 28]
```diff
@@ -34,10 +39,16 @@ export default class GoogleLogin extends Vue {
       onfailure: this.onFailure,
     });
   }
+
+  // getter to get the updated version of the instance renderLoginButton
```
Comment (TE4-oskar-pilborg) [inline frontend/src/components/GoogleLogin.vue]: new line after comment

File: frontend/src/components/LoanFormComponent.vue [hunk 29]
```diff
@@ -83,6 +83,9 @@ interface Form {
   book_id: number | string;
 }
 
+
+// loan form component is used to user interface for the user to create a loan and is later
```
Comment (TE4-oskar-pilborg) [inline frontend/src/components/LoanFormComponent.vue]: LoanFormComponent* and new line after comment

File: frontend/src/components/LoanFormComponent.vue [hunk 30]
```diff
@@ -102,6 +110,9 @@ export default class LoanFormComponent extends Vue {
     }
   }
 
+  // this is the Eventlistener for the user to reset the form if the user has entered the
```
Comment (TE4-oskar-pilborg) [inline frontend/src/components/LoanFormComponent.vue]: This*

File: frontend/src/components/NavbarComponent.vue [hunk 31]
```diff
@@ -58,6 +58,10 @@ import UsersModule from '../store/modules/UsersModule';
     GoogleLogin,
   },
 })
+
+// this is the navbarcomponent and has one single method that tries to sign out the user
```
Comment (TE4-oskar-pilborg) [inline frontend/src/components/NavbarComponent.vue]: This*

File: frontend/src/components/TitleFormComponent.vue [hunk 32]
```diff
@@ -67,6 +67,10 @@
 import { Component, Prop, Vue } from 'vue-property-decorator';
 import TitlesModule, { TitleForm } from '../store/modules/TitlesModule';
 
+
+// this is the child component of the earlier named parent element and catches the information
```
Comment (TE4-oskar-pilborg) [inline frontend/src/components/TitleFormComponent.vue]: This

File: frontend/src/components/TitleFormComponent.vue [hunk 33]
```diff
@@ -83,6 +87,9 @@ export default class TitleFormComponent extends Vue {
     { value: 'Skönlitteratur', text: 'Skönlitteratur' },
   ];
 
+  // on this eventlistener the titlesmodule recreates the form when the submit has been
```
Comment (TE4-oskar-pilborg) [inline frontend/src/components/TitleFormComponent.vue]: On* or In

File: frontend/src/components/TitleListComponent.vue [hunk 34]
```diff
@@ -12,6 +12,10 @@ import { Component, Prop, Vue } from 'vue-property-decorator';
 import TitlesModule from '../store/modules/TitlesModule';
 import { VuexModule } from 'vuex-module-decorators';
 
+
+// list component that inherits the information about the titles from the titles module
```
Comment (TE4-oskar-pilborg) [inline frontend/src/components/TitleListComponent.vue]: List*

File: frontend/src/views/AdminBooksView.vue [hunk 35]
```diff
@@ -28,6 +28,9 @@ import TitlesModule from '../store/modules/TitlesModule';
   }
 })
 export default class AdminBooksView extends Vue {
+
+  // created is the constructor of the admin books view that instantiates the class by
```
Comment (TE4-oskar-pilborg) [inline frontend/src/views/AdminBooksView.vue]: Created* and new line after comment

File: frontend/src/views/AdminCardsView.vue [hunk 36]
```diff
@@ -29,6 +35,8 @@ export default class AdminCardsViews extends Vue {
 
   public images: File[] = [];
 
+  // the on send images getter takes the imge from the event target and sets it to the instance
```
Comment (TE4-oskar-pilborg) [inline frontend/src/views/AdminCardsView.vue]: The* and new line after comment

File: frontend/src/views/AdminLoansView.vue [hunk 37]
```diff
@@ -1,3 +1,8 @@
+<!--
+    as shown in the template, this view is based of the components listed as
```
Comment (TE4-oskar-pilborg) [inline frontend/src/views/AdminLoansView.vue]: As*

File: frontend/src/views/AdminLoansView.vue [hunk 38]
```diff
@@ -26,7 +31,14 @@ import LoansModule from '../store/modules/LoansModule';
     LoanListComponent
   }
 })
+
+// this component is the parent view of the components LoanListComponent and LoanFormComponent
```
Comment (TE4-oskar-pilborg) [inline frontend/src/views/AdminLoansView.vue]: This*

File: frontend/src/views/AdminLoansView.vue [hunk 39]
```diff
@@ -26,7 +31,14 @@ import LoansModule from '../store/modules/LoansModule';
...
+// which is declared in the template
+
 export default class AdminLoansView extends Vue {
+
+  // the class is instantiated by the constructor created by fetching all the loans from the
```
Comment (TE4-oskar-pilborg) [inline frontend/src/views/AdminLoansView.vue]: The* and new line after comment

File: frontend/src/views/AdminTitlesView.vue [hunk 40]
```diff
@@ -26,7 +26,13 @@ import TitleListComponent from '@/components/TitleListComponent.vue';
     TitleListComponent
   }
 })
+
+// this component is a parent view for the components TitleFormComponent and the
```
Comment (TE4-oskar-pilborg) [inline frontend/src/views/AdminTitlesView.vue]: This* and new line after comment

File: frontend/src/views/AdminTitlesView.vue [hunk 41]
```diff
@@ -26,7 +26,13 @@ import TitleListComponent from '@/components/TitleListComponent.vue';
...
+// TitleListComponent
 export default class AdminTitlesView extends Vue {
+
+  // the created function is the constructor in vue and instantiates the class by fetching
```
Comment (TE4-oskar-pilborg) [inline frontend/src/views/AdminTitlesView.vue]: The* and new line after comment

File: frontend/src/views/AdminView.vue [hunk 42]
```diff
@@ -51,7 +55,14 @@ import { Vue, Component, Prop } from 'vue-property-decorator';
 @Component({
   components: {}
 })
+
+// this is the admin view and has no child nor parent components and is therefore only
```
Comment (TE4-oskar-pilborg) [inline frontend/src/views/AdminView.vue]: This*

File: frontend/src/views/AdminView.vue [hunk 43]
```diff
@@ -51,7 +55,14 @@ import { Vue, Component, Prop } from 'vue-property-decorator';
...
+// used for the sole purpose of giving the admin opportunities to go to the requested route
+
 export default class AdminView extends Vue {
+
+  // the items object is used to fill the tables with the right column and row information
```
Comment (TE4-oskar-pilborg) [inline frontend/src/views/AdminView.vue]: The* and new line after comment

File: backend/app/controllers/api/v1/titles_controller.rb (hunk 1 above)
Comment (TE4-Mattis-Abrahamsson) [reply backend/app/controllers/api/v1/titles_controller.rb]: #wontfix ?

Comment (Te4-Alexander-Nilsson-Melbostad) [reply backend/app/controllers/api/v1/titles_controller.rb]: #willfix

File: frontend/src/components/BookFormComponent.vue (hunk 12 above)
Comment (Te4-Alexander-Nilsson-Melbostad) [reply frontend/src/components/BookFormComponent.vue]: I do not agree with this since the phrase is not a name

File: backend/app/policies/loan_policy.rb [hunk 44]
```diff
@@ -31,7 +31,7 @@ def update?
   # person who created the loan in order to see the information about it
 
   def show?
-    user&.admin? || user&.uid == record&.lent_by_id
+    user&.admin? || user&.uid == record&.lent_by_id || user&.uid == record&.loaned_by_id
```
Comment (TE4-jacob-lundberg) [inline backend/app/policies/loan_policy.rb]: Is this included in the scope of the issue? Changing a policy in documentation issue?

Comment (Te4-Alexander-Nilsson-Melbostad) [reply backend/app/policies/loan_policy.rb]: Nope you are correct and it is not acceptable! Will revert the changes and create a pull request from the policy branch, thanks for noticing.

File: frontend/src/components/CigCanvas.vue [hunk 45]
```diff
@@ -100,21 +121,33 @@ export default class CigCanvas extends Vue {
...
+  // the structure of the canvas
+
   public getCanvasContainerSize(): void {
     const canvasContainer = this.$refs.canvasContainer as HTMLDivElement;
+
     this.width = canvasContainer.clientWidth;
     this.height = canvasContainer.clientHeight;
   }
 
+  // the setCanvasSize method basically does the same thing but sets the dimensions to 2d
```
Comment (TE4-oskar-pilborg) [inline frontend/src/components/CigCanvas.vue]: The* 💋

File: frontend/src/components/CigCanvas.vue [hunk 46]
```diff
@@ -100,21 +121,33 @@ export default class CigCanvas extends Vue {
...
     const htmlCanvasElement = this.$refs.canvas as HTMLCanvasElement;
+
     this.context = htmlCanvasElement.getContext('2d');
 
     htmlCanvasElement.width = this.width;
     htmlCanvasElement.height = this.height;
   }
 
+
+  // the draw image instantiates new classes of the image based on the barcode and also
```
Comment (TE4-oskar-pilborg) [inline frontend/src/components/CigCanvas.vue]: The* 💋

File: frontend/src/components/CigCanvas.vue [hunk 47]
```diff
@@ -171,7 +204,12 @@ export default class CigCanvas extends Vue {
     }, 200);
   }
 
+
+  // draw text method renders the text on the canvas in order to style it and also get
```
Comment (TE4-oskar-pilborg) [inline frontend/src/components/CigCanvas.vue]: Draw* 💋

File: frontend/src/components/CigCanvas.vue [hunk 48]
```diff
@@ -201,7 +246,12 @@ export default class CigCanvas extends Vue {
     });
   }
 
+
+  // the download canvas method is used to download and also save the donwloaded zip to
```
Comment (TE4-oskar-pilborg) [inline frontend/src/components/CigCanvas.vue]: The* 💋

File: frontend/src/components/CigCanvas.vue [hunk 49]
```diff
@@ -224,12 +280,21 @@ export default class CigCanvas extends Vue {
...
+  // and the generate Canvas 
+
   public onNameInput(): void {
+
     this.checkUserData();
     this.generateCanvas();
   }
 
+
+  // this is also a getter to receive information simply from the generate canvas instance
```
Comment (TE4-oskar-pilborg) [inline frontend/src/components/CigCanvas.vue]: This* 💋

File: frontend/src/components/LoanFormComponent.vue [hunk 50]
```diff
@@ -61,6 +61,8 @@ interface Form {
   book_id: number | string;
 }
 
+// loan form component is used to user interface for the user to create a loan and is later
```
Comment (TE4-oskar-pilborg) [inline frontend/src/components/LoanFormComponent.vue]: LoanFormComponent maybe?

File: frontend/src/components/LoanFormComponent.vue [hunk 51]
```diff
@@ -80,7 +87,12 @@ export default class LoanFormComponent extends Vue {
     }
   }
 
+
+  // this is the Eventlistener for the user to reset the form if the user has entered the
```
Comment (TE4-oskar-pilborg) [inline frontend/src/components/LoanFormComponent.vue]: This* 💋

File: backend/app/controllers/api/v1/titles_controller.rb [hunk 52]
```diff
@@ -38,11 +53,17 @@ def update
...
     @title.destroy
     render json: @title
   end
 
+  # Set title method, find the current title and sets the title to that instance in order
+  # to execute given method on correct title
+
   private
     # Use callbacks to share common setup or constraints between actions.
     def set_title
```
Comment (TE4-Mattis-Abrahamsson) [inline backend/app/controllers/api/v1/titles_controller.rb]: In this file, you are writing "JSON", when in previous files you wrote "json", it could be nice with some consistency. Not a big issue though.

File: frontend/src/views/AdminCardsView.vue [hunk 53]
```diff
@@ -27,7 +33,10 @@ import UsersModule from '../store/modules/UsersModule';
   }
 })
 export default class AdminCardsViews extends Vue {
-  private images: File[] = [];
+  public images: File[] = [];
+
```
Comment (TE4-Mattis-Abrahamsson) [inline frontend/src/views/AdminCardsView.vue]: Why did you change this?

Comment (Te4-Alexander-Nilsson-Melbostad) [reply frontend/src/views/AdminCardsView.vue]: I did not intentionally changed this but was apparently changed when i tried to fix the merge conflicts! will revert the changes!

File: backend/app/controllers/api/v1/titles_controller.rb (hunk 52 above)
Comment (Te4-Alexander-Nilsson-Melbostad) [reply backend/app/controllers/api/v1/titles_controller.rb]: Fixed this now, thanks!

File: frontend/src/views/AdminCardsView.vue (hunk 53 above)
Comment (te4-Leo-Wahlandt) [reply frontend/src/views/AdminCardsView.vue]: is this resolved?
